src/org/daisy/dotify/formatter/impl/core/TableOfContentsImpl.java

```diff
@@ -132,12 +132,19 @@ public void endEntry() {
      * of the way the sequence of blocks is constructed, we are potentially throwing away borders
      * and margins that should be kept. That said, the previous implementation did not handle
      * borders and margins correctly either, so fixing this issue can be seen as an optimization.</p>
+     * 
+     * <p>This method sets the meta page number of the range blocks that are filtered out.</p> 
```

Comment:
"sets the meta page number of ... _to the provided value_." ?

Reply:
@PaulRambags Have you seen above comment? Minor thing, just so that there can not be confusion about who provides the values (i.e. the caller, not the function).

Reply:
Done.